dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/ingest/tabulardata/impl/plugins/dta/DTAFileReader.java

```diff
@@ -529,7 +531,7 @@ public TabularDataIngest read(BufferedInputStream stream, File dataFile) throws
 
             ingesteddata.setDataTable(dataTable);
         } catch (IllegalArgumentException iaex) {
-            throw new IOException(iaex.getMessage());
+            throw new IngestException(IngestError.UNKNOWN_ERROR, iaex);
```

Comment:
Are you sure you want to pass `iaex.toString()` as argument, while it was `iaex.getMessage()`?
Alternatively, why not allow `IngestException` to accept a causing exception and handle that cause message as argument in there?

Reply:
It is actually passed as `Throwable cause` here.

Reply:
Yes, it is, you're right.